tools/oeutil/host/generate_evidence.cpp

```diff
@@ -221,11 +221,9 @@ static void oeutil_quote_provider_log(
 // Set DCAP client (libdcap_quoteprov) log callback
 static void set_log_callback()
 {
-#if defined(__linux__)
     // Initialize quote provider and set log callback
     oe_initialize_quote_provider();
     oe_sgx_set_quote_provider_logger(oeutil_quote_provider_log);
```

Comment:
Should the oeutil_quote_provider_log be updated as well?

Reply:
Thanks Mingwei! Updated oeutil_quote_provider_log().